app/carbonapi/http_handlers.go

```diff
@@ -617,7 +620,7 @@ func (app *App) findHandler(w http.ResponseWriter, r *http.Request) {
 
 	logAsError := false
 	defer func() {
-		deferredAccessLogging(r, &accessLogDetails, t0, logAsError)
+		app.deferredAccessLogging(r, &accessLogDetails, t0, logAsError)
```

Comment:
It can always be called simply as:
defer app.deferredAccessLogging(...), or, as I suggested earlier: defer deferredAccessLogging(app, ...)

Reply:
Yes, you are 100% right. This needs to be cleaned up altogether though. This will be taken care of in one of the following refactorings.

Thanks for pointing this out.